tensorflow/examples/skflow/mnist_rnn.py

```diff
@@ -0,0 +1,76 @@
+#  Copyright 2015-present The Scikit Flow Authors. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+"""
+This example builds rnn network for mnist data.
+Borrowed structure from here: https://github.com/aymericdamien/TensorFlow-Examples/blob/master/examples/3%20-%20Neural%20Networks/recurrent_network.py
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from sklearn import metrics, preprocessing
+
+import tensorflow as tf
+from tensorflow.contrib import learn
+from tensorflow.examples.tutorials.mnist import input_data
+
+# Parameters
+learning_rate = 0.1
+training_steps = 3000
+batch_size = 128
+
+# Network Parameters
+n_input = 28  # MNIST data input (img shape: 28*28)
+n_steps = 28  # timesteps
+n_hidden = 128  # hidden layer num of features
+n_classes = 10  # MNIST total classes (0-9 digits)
+
+### Download and load MNIST data.
+mnist = input_data.read_data_sets("/tmp/data/")
```

Comment:
my tensorflow version is 0.8.0,   learn.datasets.mnist.read_data_sets("/tmp/data") or learn.datasets.load_mnist() doesn't work.